### DISCUSSION TITLE
Feature Request: Support for non-hive style partitioning

### DISCUSSION THREAD

**Discussion by MatinF** [duckdb/duckdb#9951] [Ideas]
We use Parquet data lakes with a structure as below:

```
TABLEX/YYYY/MM/DD/XYZ.parquet
```
We are unable to setup the partitioning in hive style, but would still like to leverage partition filters in DuckDB. In Amazon Athena this can be done within the AWS Glue table settings by specifying the syntax for the date structure. This enables queries as below:

```
SELECT
    *
FROM
    TABLEX
WHERE
    date_created BETWEEN '2020/1/1' AND '2020/10/10'
```

Basically I am looking for how to create an equivalent of date_created in DuckDB - or some other way of leveraging my existing partitioning structure.

**Comment by sveniu** [comment]
Is this covered by https://github.com/duckdb/duckdb/discussions/6897?

**Reply by MatinF** [reply]
Yes, that seems identical to our requirement
